zerver/views/muting.py

```diff
@@ -64,14 +64,17 @@ def unmute_topic(
     do_unmute_topic(user_profile, stream, topic_name)
 
 
+VALID_UPDATE_MUTED_TOPIC_OPTIONS = ["add", "remove"]
```

Comment:
The “op” here is for “operation”, not “option”.

I’d probably just inline this. The name is longer than the value, and doesn’t convey any information that the context doesn’t.

Reply:
Updated.